doc/sphinx/language/gallina-specification-language.rst

```diff
@@ -520,8 +646,32 @@ Sections :ref:`if-then-else` and :ref:`irrefutable-patterns`).
    single: fix
    single: cofix
 
-Recursive functions
--------------------
+Recursive and co-recursive functions: fix and cofix
+---------------------------------------------------
+
+.. insertgram term_fix term1_extended_opt
+
+.. productionlist:: coq
+   term_fix : `single_fix`
+            : `single_fix` with `fix_bodies` for `ident`
+   single_fix : fix `fix_body`
+              : cofix `fix_body`
+   fix_bodies : `fix_bodies` with `fix_body`
+              : `fix_body`
+   fix_body : `ident` `binders_opt` `fixannot_opt` `colon_term_opt` := `term`
+   binders_opt : `binders`
```

Comment:
Sorry, I have one last request. Can you move the definition of `binders_opt` in the binders section (just above `binders` I guess). It is now referenced from the main term grammar chunk.